### ISSUE TITLE
Choose a Google Drive dir

### ISSUE THREAD

**Issue by HanneMaes** [anthonygarvan/marktwo#40]
I have a folder in Google Drive with all my markdown files.
It would be very useful if I could point marktwo to that directory and edit the files.

**Comment by cbertelli** [comment]
Yes, folders are more useful for different topics than for different file types. Unfortunately it would be difficult to distinguish markdown files in several folders and it would need adding a "file navigator". I would like it very much if possible.
